queries/bibtex/indents.scm

```diff
@@ -1,10 +1,10 @@
 [
  (entry)
-] @indent
+] @indent.begin.begin
```

Comment:
wrong capture here

Reply:
Thanks for catching these.  Did a quick additional check for "indent.begin.".  I thought i'd cleaned these up, but obv. missed a few.